docs/lightning/04-salesforce-lambdas/04-contact-channel-analytics.md

```diff
@@ -265,6 +265,26 @@ Please note that your secret may also be formatted stored as a "Secret key/value
 12. NOTE: The recording playback, waveform, and transcript views are
     only active when you also choose to activate recording transcripts.
 
+#### Common Audio Streaming Setup Issues
+
+1. Verify that the Secrets Manager secret contains both the `CloudFrontPrivateKey` and `CloudFrontAccessKeyID` items.
+
+2. Verify that your Cloudfront distribution's behavior is set to use `Trusted Key Groups`, and that the correct Key Group is selected.
+
+<img src={useBaseUrl('/img/shared/audiostreamingkeygroups.png')} />
+
+3. Verify that your Cloudfront distribution's behavior contains the sfSig4RequestToS3 edge lambda
+
+<img src={useBaseUrl('/img/shared/audiostreamingedgelambda.png')} />
+
+4. Verify that your S3 bucket CORS configuration is correct
```

Comment:
Maybe we should add that they may need to add `https://xxxx.lightning.force.com` or `https://xxxx.my.salesforce.com` to their CORS config.

Reply:
do you know under what conditions they need to do this though?

Reply:
No, I haven't figured out the conditions, I just figured it might be worth mentioning in the troubleshooting, which may reduce the number of tickets we get.

Reply:
spoke to adi offline - going to investigate this further in a separate PR